### DISCUSSION TITLE
questions

### DISCUSSION THREAD

**Discussion by omerboyarlar** [IBAX-io/go-ibax#45] [Q&A] [answered]
I did what is done in the video on your page to create a wallet. but on the telegram page it says we need to save the private key. but I can't login again. Will all my work and references be lost? What can I do.
(I saved the keywords but it doesn't work)
Why didn't you say that in the video?

**Comment by HalloKyle** [comment, marked as answer]
Hi you can also check this url for solution: https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by hemresenol** [comment]
contact customer service and they will fix it
